rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/logsdb/10_settings.yml

```diff
@@ -0,0 +1,394 @@
+                type: text
+
+  - do:
+      indices.get_settings:
+        index: test-sort-order
+
+  - is_true: test-sort-order
+  - match: { test-sort-order.settings.index.mode: "logs" }
+  - match: { test-sort-order.settings.index.sort.field.0: null }
+  - match: { test-sort-order.settings.index.sort.field.1: null }
+  - match: { test-sort-order.settings.index.sort.order.0: "asc" }
+  - match: { test-sort-order.settings.index.sort.order.1: "asc" }
+
+---
+override sort missing settings:
+  - requires:
+      cluster_features: ["gte_v8.15.0"]
+      reason: introduced in 8.15.0
+
+  - do:
+      indices.create:
+        index: test-sort-missing
+        body:
+          settings:
+
+            index:
+              mode: logs
+              number_of_shards: 2
+              number_of_replicas: 0
+              sort:
+                missing: [ "_last", "_first" ]
+          mappings:
+            properties:
+              "@timestamp":
+                type: date
+              hostname:
+                type: keyword
+              agent_id:
+                type: keyword
+              process_id:
+                type: integer
+              http_method:
+                type: keyword
+              message:
+                type: text
+
+  - do:
+      indices.get_settings:
+        index: test-sort-missing
+
+  - is_true: test-sort-missing
+  - match: { test-sort-missing.settings.index.mode: "logs" }
+  - match: { test-sort-missing.settings.index.sort.field.0: null }
+  - match: { test-sort-missing.settings.index.sort.field.1: null }
+  - match: { test-sort-missing.settings.index.sort.missing.0: "_last" }
+  - match: { test-sort-missing.settings.index.sort.missing.1: "_first" }
+
+---
+override sort mode settings:
+  - requires:
+      cluster_features: ["gte_v8.15.0"]
+      reason: introduced in 8.15.0
+
+  - do:
+      indices.create:
+        index: test-sort-mode
+        body:
+          settings:
+
+            index:
+              mode: logs
+              number_of_shards: 2
+              number_of_replicas: 0
+              sort:
+                mode: [ "max", "max" ]
+          mappings:
+            properties:
+              "@timestamp":
+                type: date
+              hostname:
+                type: keyword
+              agent_id:
+                type: keyword
+              process_id:
+                type: integer
+              http_method:
+                type: keyword
+              message:
+                type: text
+
+  - do:
+      indices.get_settings:
+        index: test-sort-mode
+
+  - is_true: test-sort-mode
+  - match: { test-sort-mode.settings.index.mode: "logs" }
+  - match: { test-sort-mode.settings.index.sort.field.0: null }
+  - match: { test-sort-mode.settings.index.sort.field.1: null }
+  - match: { test-sort-mode.settings.index.sort.mode.0: "max" }
+  - match: { test-sort-mode.settings.index.sort.mode.1: "max" }
+
+---
+override sort field using nested field type:
```

Comment:
Should we also have a test that uses nested field type and uses default logs index mode sorting?